docs/other-operators.md

```diff
@@ -25,6 +25,38 @@ __Example__
 
 `Price < 50 ? "Cheap" : "Expensive"`
 
+## `?:` (Default/Elvis)
+
+The default (or "elvis") operator returns the left-hand side if it is truthy, otherwise it returns the right-hand side. This is useful for providing fallback values when an expression may evaluate to a falsy value (such as `null`, `false`, `0`, `''`, or `undefined`).
```

Comment:
This is correct, although I use the term 'effective Boolean value' rather than truthy / falsy. (this comes from the terminology used in XPath / XQuery).
Perhaps also you could add some words on this [here](https://docs.jsonata.org/next/programming#conditional-logic) too.